packages/cli/src/commands/domains/index.ts

```diff
@@ -173,8 +200,8 @@ www.example.com  CNAME            www.example.herokudns.com
       if (customDomains && customDomains.length > 0) {
         ux.stdout()
 
-        if (customDomains.length > 100 && !flags.json) {
-          ux.warn(`This app has over 100 domains. Your terminal may not be configured to display the total amount of domains. You can output domains in JSON format with: ${color.cmd('heroku domains -a example-app --json')}`)
+        if (customDomains.length > 100 && !flags.json && !flags.csv) {
+          ux.warn(`This app has over 100 domains. Your terminal may not be configured to display the total amount of domains. You can export all domains into a CSV file with: ${color.cmd('heroku domains -a example-app --csv > example-file.csv')}`)
```

Comment:
We should leave the JSON copy as is because that was the initial flow for users with over 100 domains. The previous functionality still had CSV formatting available, but wasn't highlighted in the warning. They also can view that format option in the `--help` CSV flag description.